Dockerfile

```diff
@@ -22,6 +22,7 @@ WORKDIR /app
 COPY package*.json ./
 RUN npm ci --only=production
 COPY utils/ ./utils
+COPY config/ ./config
```

Comment:
All those copies can just be `COPY . .` with a correct `.dockerignore` instead?

Reply:
I had this before, but easier is to copy this what is just needed.

Reply:
It follows that if we only build one image, "what is just needed" refers to _everything_, because what is not needed should be excluded by the .dockerignore anyways.